### PR TITLE
Dod update

### PR DESCRIPTION
**Pull request by rachelschneiderman**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ustc-story-template.md [hunk 1]
```diff
@@ -26,8 +26,7 @@ assignees: ''
 
 ## Tasks
 
-## Definition of Done (Updated 10-26-20)
-
+## Definition of Done (Updated 2-23-20)
```
Comment (rachaelparris) [inline .github/ISSUE_TEMPLATE/ustc-story-template.md]: `2-23-21` 😄

File: .github/ISSUE_TEMPLATE/ustc-story-template.md [hunk 2]
```diff
@@ -53,4 +53,4 @@ assignees: ''
  - [ ] Rebuild entity documentation
  - [ ] Acceptance criteria for the story has been met
  - [ ] Deployed to the dev environment
- - [ ] Deployed to the prod environment
+ - [ ] Deployed to the prod environment
```
Comment (rachaelparris) [inline .github/ISSUE_TEMPLATE/ustc-story-template.md]: Should we go ahead and update this to the court's migration environment?

File: .github/ISSUE_TEMPLATE/ustc-story-template.md (hunk 1 above)
Comment (rachelschneiderman) [reply .github/ISSUE_TEMPLATE/ustc-story-template.md]: Alt Text

File: .github/ISSUE_TEMPLATE/ustc-story-template.md (hunk 2 above)
Comment (rachelschneiderman) [reply .github/ISSUE_TEMPLATE/ustc-story-template.md]: I was debating that myself! I vote yes.

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**